src/components/ec-main-container/ec-main-container.spec.js

```diff
@@ -31,6 +31,17 @@ describe('EcMainContainer', () => {
     expect(wrapper.element).toMatchSnapshot();
   });
 
+  it('should display only the title when the titleIntro is not given', () => {
```

Comment:
I like to write the specs on top of each other, thus I'd phrase this as "it renders properly" then a "it renders with titleIntro"